wire-ios-share-engine/Sources/OperationLoop.swift

```diff
@@ -158,9 +158,9 @@ final class OperationLoop: NSObject, RequestAvailableObserver {
 
     func setupObserver(for context: NSManagedObjectContext, onSave: @escaping SaveClosure) -> NSObjectProtocol {
         return NotificationCenter.default.addObserver(forName: .NSManagedObjectContextDidSave, object: context, queue: callBackQueue) { note in
-            if let insertedObjects = note.userInfo?[NSInsertedObjectsKey] as? Set<NSManagedObject>, let updatedObjects = note.userInfo?[NSUpdatedObjectsKey] as? Set<NSManagedObject> {
-                onSave(note, insertedObjects, updatedObjects)
-            }
+          let insertedObjects = (note.userInfo?[NSInsertedObjectsKey] as? Set<NSManagedObject>) ?? Set<NSManagedObject>()
```

Comment:
I've noticed that we have exactly the same case for NotificationEngine's OperationLoop that may cause similar issues in the future for notifications.
What do you think about applying this change there as well?

Reply:
We already discussed that with @johnxnguyen! I'm planning to open a PR with those changes as well. @netbe